code/online_challenges/src/hackerrank/3D_aurface_area/README.md

```diff
@@ -0,0 +1,3 @@
+# Problem Link
+[3D_aurface_area](https://www.hackerrank.com/challenges/3d-surface-area/problem?h_r=internal-search)
```

Comment:
Could you add more detail to each file? Perhaps similar to what we do for the project euler problems.